--- a/res/config.ini	2025-08-06 14:07:35.182697343 +0200
+++ b/res/config.ini	2025-08-06 14:07:15.694850289 +0200
@@ -174,7 +174,7 @@
 path = /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
 
 # Command executed when pressing restart_key
-restart_cmd = /sbin/shutdown -r now
+restart_cmd = /usr/bin/loginctl reboot
 
 # Specifies the key used for restart (F1-F12)
 restart_key = F2
@@ -190,19 +190,19 @@
 # By default it's saved in the user's home directory
 # Important: due to technical limitations, X11 and shell sessions aren't supported, which
 # means you won't get any logs from those sessions
-session_log = ly-session.log
+session_log = .ly-session.log
 
 # Setup command
 setup_cmd = $CONFIG_DIRECTORY/ly/setup.sh
 
 # Command executed when pressing shutdown_key
-shutdown_cmd = /sbin/shutdown -a now
+shutdown_cmd = /usr/bin/loginctl poweroff
 
 # Specifies the key used for shutdown (F1-F12)
 shutdown_key = F1
 
 # Command executed when pressing sleep key (can be null)
-sleep_cmd = null
+sleep_cmd = /usr/bin/loginctl suspend
 
 # Specifies the key used for sleep (F1-F12)
 sleep_key = F3
